dotnet/content/install.cmd

```diff
@@ -1,7 +1,20 @@
+
+REM Entrypoint: https://github.com/projectkudu/kudu/blob/master/Kudu.Core/SiteExtensions/SiteExtensionManager.cs#L548
+
 @echo off
 
 echo Starting install
 
+IF EXIST ".\applicationHost.xdt" (
+  echo Upgrade will not apply until full application stop
+)
+
 POWERSHELL .\install.ps1
 
+IF EXIST ".\applicationHost.xdt" (
```

Comment:
This doesn't make sense to me. If we are upgrading and we apply the new `applicationHost.xdt`, that should count as a success but it looks like it fails instead

Reply:
Oh, this probably replaced the presence of the `.\installation-failure.txt`

Reply:
Yeah, this was an error, it's fixed. I introduced this when switching to this mechanism and made a lot of changes locally.